content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
+- **Run metric change threshold met**: Apply recurring actions based on whether a run metric's amount of change (absolute or relative) is above, below, or exactly equal to the threshold. Optionally, specify how to average results.
+
+## Evant actions
+An automation can run a webhook on a third-party service such as GitHub or Microsoft Teams, or it can post to a Slack channel.
+
+## Create an automation
+These sections show how to configure a Slack automation or a webhook automation.
+
+### Configure a Slack automation
+Configuring a Slack integration takes multiple steps. First, [add one or more Slack channels]({{< relref "#add-a-slack-channel" >}}) as alert destinations. Next, [create the automation that notifies the Slack channel]({{< relref "#create-slack-automation" >}}).
+
+#### Add a Slack channel
+
+1. Go to Team Settings page.
+1. To integrate with a new Slack workspace and channel, click **Connect Slack**.
+    {{% alert %}}
+    To integrate with a new Slack channel in a workspace that is already set up, click **New integration**.
+    {{% /alert %}}
+
+  If necessary, sign in to your Slack instance. A browser window appears, asking that you grant Weights and Biases permission to post to the Slack channel you select. Read the page, then click **Search for a channel** and begin typing the channel name. Select the channel from the list, then click **Allow**.
+
+1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: Weights & Biases`, the integration is configured correctly.
+
+Now you can [create an automation that uses the Slack integration]({{< relref "#create-slack-automation" >}}).
+
+#### Create the automation {#create-slack-automation}
+
+After you [configure a Slack integration]({{< relref "#configure-the-slack-integration" >}}), follow these steps to create an automation that uses it.
+
+1. Go to the project page.
+1. In the sidebar, click **Automations**.
+1. Click **Create automation**.
+1. Choose the **Event** which triggers the automation. If applicable, provide options that are specific to the event type. If your project has no registries, registry events will not be available. Click **Next step**.
+1. Select the team where you added the Slack integration.
+1. Set **Action type** to **Slack notification**. Select the Slack channel, then click **Next step**.
+1. Proviude a name for the automation. Optionally, provide a description.
+1. Click **Create automation**.
+
+### Configure a webhook automation
+
+Configuring a webhook integration takes multiple steps:
+
+1. If your webhook requires any sensitive strings, [add them as secrets]({{< relref "#add-a-secret" >}}) for any sensitive strings required by the webhook. If the webhook requires a bearer token, do not configure a secret for it. Instead, you configure it as part of creating the webhook.
```

Comment:
you repeat the "any sensitive strings" phrase

Reply:
Based on the current UI, I believe bearer tokens are also pulled from secrets

Reply:
Nope, it's a separate Access Token field.

Reply:
yes, it's a separate field, but the list you're selecting from (for both Secret and Access Token fields) is the list of secrets.

Note that when you open the dropdown for both access token and secret fields, it's the same list of stuff 
![image](https://github.com/user-attachments/assets/93318023-b6fb-4b68-807d-f4a2d64340c8)
![image](https://github.com/user-attachments/assets/50f49b8b-bb4e-4584-bee6-49e58c382dc2)

Reply:
Done!